### PR TITLE
[DC-773] Fix query variable in create_deid_map from dict to list

### PR DESCRIPTION
**Pull request by amcgrenera-vumc**

### CODE REVIEW

File: data_steward/retraction/create_deid_map.py [hunk 1]
```diff
@@ -142,10 +142,10 @@ def run_queries(queries):
     :param queries: list of queries
     """
     query_job_ids = []
-    for query_dict in queries:
+    for query in queries:
         logging.info('Creating or renaming _deid_map table with query: %s' %
-                     (query_dict['query']))
-        job_results = bq_utils.query(q=query_dict['query'], batch=True)
+                     (query))
```
Comment (dcampbell-vumc) [inline data_steward/retraction/create_deid_map.py:147]: nit: The redundant parenthesis can be removed.
```suggestion
                     query)
```

**Review by dcampbell-vumc — CHANGES_REQUESTED** [review]
Just a little nit pick change and then this will be good to go!

File: data_steward/retraction/create_deid_map.py (hunk 1 above)
Comment (amcgrenera-vumc) [reply data_steward/retraction/create_deid_map.py]: good catch, thank you! I've taken it out!

**Comment by amcgrenera-vumc** [comment]
@dcampbell-vumc the integration test is failing, I'm going to investigate.

**Comment by lrwb-aou** [comment]
> @dcampbell-vumc the integration test is failing, I'm going to investigate.

@amcgrenera-vumc and @dcampbell-vumc This is failing because of [Line 173 in retract_deactivated_pids.py](https://github.com/all-of-us/curation/blob/develop/data_steward/retraction/retract_deactivated_pids.py#L173) and the branch name.  Take `deid` out of your branch name and it will work just fine.  Line 173 is looking for either `deid` in the dataset name or a `de_identified: true` dataset label.  Test datasets are built using your branch name.  So, it's hitting this and failing because the test isn't providing a pid_rid mapping table name.

**Pull request closed without merging**